daemon/lokinet-vpn.cpp

```diff
@@ -0,0 +1,419 @@
+#include <lokimq/lokimq.h>
+#include <nlohmann/json.hpp>
+#include <cxxopts.hpp>
+#include <future>
+#include <vector>
+#include <array>
+
+#ifdef _WIN32
+// add the unholy windows headers for iphlpapi
+#include <winsock2.h>
+#include <ws2tcpip.h>
+#include <iphlpapi.h>
+#include <stdio.h>
+
+#pragma comment(lib, "iphlpapi.lib")
```

Comment:
who uses MSVC

Reply:
i copy pasted this from the windows documentation